java/main/src/test/java/com/saucelabs/saucebindings/budget.yml

```diff
@@ -0,0 +1,6 @@
+"https://www.saucedemo.com":
```

Comment:
@titusfortner can you please explain what this is and it's intended usage?

Reply:
So, for JS they use JSON; I wanted to implement something that uses a Map, so you can convert either yaml or json into the map it will work. I didn't want to use JSON because I don't want to deal with Jackson library, yet. :) More info: https://docs.saucelabs.com/performance/transitions#defining-a-performance-budget

Reply:
@titusfortner I worry that this feature is unnecessarily making the code complicated and overengineered. From my experience, customers aren't advanced enough to handle such a use case (or even me 😂). Instead they are only interested in [3 use cases](https://github.com/saucelabs-training/demo-java/blob/master/selenium-junit4-examples/src/test/java/com/saucedemo/PerformanceExampleTests.java). Specifically they like `multiPagePerformanceExample()` and the ` simplePerformanceTest()`

Reply:
Yeah, We have docs for it with code examples, so I thought we should provide support for it. I'm not sure how much it would be used. I had some thoughts on expanding it later, but definitely not the thing that should be holding us up.

This is the kind of feature I wouldn't make an issue out of in the Sauce Bindings Documentation, but would have example code in the Sauce Docs with it for people who are interested in playing with it. As such, the confusion factor of the average user shouldn't really matter. It's the same kind of thing with the `mergeCapabilities()` in SauceOptions.